app/src/main/java/com/github/braillesystems/learnbraille/data/db/LearnBrailleDatabase.kt

```diff
@@ -209,119 +210,132 @@ private val MIGRATION_17_18 = object : Migration(17, 18) {
-                )
-            }
+                }
+            )
+        }
 
-            materials?.forEach {
-                database.insert(
-                    "materials",
-                    SQLiteDatabase.CONFLICT_ABORT,
-                    it.run {
-                        ContentValues().apply {
-                            put("id", id)
-                            put("data", MaterialDataTypeConverters().to(data))
-                        }
+        materials?.forEach {
+            database.insert(
+                "materials",
+                SQLiteDatabase.CONFLICT_ABORT,
+                it.run {
+                    ContentValues().apply {
+                        put("id", id)
+                        put("data", MaterialDataTypeConverters().to(data))
                     }
-                )
-            }
+                }
+            )
+        }
 
-            stepAnnotations?.forEach {
-                database.insert(
-                    "step_annotations",
-                    SQLiteDatabase.CONFLICT_ABORT,
-                    it.run {
-                        ContentValues().apply {
-                            put("id", id)
-                            put("name", name)
-                        }
+        stepAnnotations?.forEach {
+            database.insert(
+                "step_annotations",
+                SQLiteDatabase.CONFLICT_ABORT,
+                it.run {
+                    ContentValues().apply {
+                        put("id", id)
+                        put("name", name)
                     }
-                )
-            }
+                }
+            )
+        }
 
-            stepsHasAnnotations?.forEach {
-                database.insert(
-                    "step_has_annotations",
-                    SQLiteDatabase.CONFLICT_ABORT,
-                    it.run {
-                        ContentValues().apply {
-                            put("course_id", courseId)
-                            put("lesson_id", lessonId)
-                            put("step_id", stepId)
-                            put("annotation_id", annotationId)
-                        }
+        stepsHasAnnotations?.forEach {
+            database.insert(
+                "step_has_annotations",
+                SQLiteDatabase.CONFLICT_ABORT,
+                it.run {
+                    ContentValues().apply {
+                        put("course_id", courseId)
+                        put("lesson_id", lessonId)
+                        put("step_id", stepId)
+                        put("annotation_id", annotationId)
                     }
-                )
-            }
+                }
+            )
         }
+    }
+
+    Timber.i("New data inserted")
+}
 
+private val MIGRATION_18_19 = object : Migration(18, 19) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        Timber.i("Start 18-19 migration")
+        updateTheoryAndMaterials(database = database)
```

Comment:
It is not needed here to provide named arguments

Reply:
fixed